docs/concepts/models.md

```diff
@@ -723,19 +643,23 @@ except ValidationError as e:
     """
 ```
 
-If you set the `model_config` or make use of `@field_validator` or other Pydantic decorators in your generic model
-definition, they will be applied to parametrized subclasses in the same way as when inheriting from a `BaseModel`
-subclass. Any methods defined on your generic class will also be inherited.
+1. Refers to the step 1 described above.
```

Comment:
```suggestion
1. Refers to step 1 described above.
```

And likewise for the others. Great idea to have annotations here (that numerically correspond, nicely)